internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -812,6 +814,9 @@ func (c *ClusterManager) buildResourceSet(cluster api.Cluster) types.ResourceSet
 	if s := c.buildImagePullSecret(kasFleetshardNamespace); s != nil {
 		r = append(r, s)
 	}
+	if s := c.buildImagePullSecret(observabilityNamespace); s != nil {
```

Comment:
Is the image pull secret content valid to pull images from the repository where the observability image proxy will be located? The `buildImagePullSecret` method always reads from the same file and therefore from the same set of image pull credentials

Reply:
The image is in the rhoas org in quay. We have tested the image pull secret and were able to pull with it. @vmanley when you tested it, did you copy the image pull secret from one of the other namespaces?